apps/src/templates/studioHomepages/ContentBox.jsx

```diff
@@ -48,7 +49,7 @@ const styles = {
   }
 };
 
-const CollapsibleSection = React.createClass({
+const ContentBox= React.createClass({
```

Comment:
Box implies a visual box.  I'd recommend a name ending in Container to better express what this does (unless it really draws a box?)